### ISSUE TITLE
Package module tip

### ISSUE THREAD

**Issue by MaykerStudio** [Timps/TimpsUnityTools#2]
Hi,you could change this project to the Unity's package pattern, so it'll be more visible and easy to manager.

**Comment by Timps** [comment]
Yeah, I definitely can.
Do you mean just to bundle it up as a package @MaykeFrankley ?
I can do that 100% for the next release.

**Comment by MaykerStudio** [comment]
Yes. Unity recently posted a tutorial of how to create custom packages, it's kinda easy.
And you could put into the Asset Store too.
